ansible/roles/packages/tasks/redhat.yaml

```
@@ -2,20 +2,20 @@
 - name: install common RPMS
   yum:
     name:
-      - audit
+      - "{{ 'python3-' if ansible_distribution_major_version == '8' }}audit"
       - ca-certificates
       - conntrack-tools
       - chrony
       - curl
-      - ebtables
+      - "{{ 'iptables-' if ansible_distribution_major_version == '8' }}ebtables"
       - open-vm-tools
       - python3-pip
       - "python{{ '3' if ansible_distribution_major_version == '8' }}-netifaces"
       - "python{{ '3' if ansible_distribution_major_version == '8' }}-requests"
       - socat
       - sysstat
       - nfs-utils
-      - NetworkManager
+      - "NetworkManager{{ '-cloud-setup' if packer_builder_type not in ['vsphere'] }}" # OS images on cloud platforms (AWS, AZURE, GCP) supports only NetworkManager-cloud-setup package.
```

https://teamcity.mesosphere.io/buildConfiguration/ClosedSource_KonvoyImageBuilder_E2eRhel82offlineFips/3646884?buildTab=log&focusLine=3157&linesState=523.2639

>: No package matching 'NetworkManager-cloud-setup' found available, maybe something like?

```suggestion
      - "NetworkManager{{ '-cloud-setup' if packer_builder_type not in ['vsphere'] and  ansible_distribution_major_version == '8' }}" # OS images on cloud platforms (AWS, AZURE, GCP) supports only NetworkManager-cloud-setup package.
```

---

fixed